DNN Platform/Library/Entities/Users/Profile/ProfilePropertyAccess.cs

```diff
@@ -243,7 +243,7 @@ public static string GetRichValue(ProfilePropertyDefinition property, string for
                         break;
                     case "richtext":
                         var objSecurity = new PortalSecurity();
-                        result = PropertyAccess.FormatString(objSecurity.InputFilter(HttpUtility.HtmlDecode(property.PropertyValue), PortalSecurity.FilterFlag.NoScripting), formatString);
+                        result = PropertyAccess.FormatString(objSecurity.InputFilter(property.PropertyValue, PortalSecurity.FilterFlag.NoScripting), formatString);

```

Comment:
Biography can be added bu any registered user. Therefore, allowing users to add these kinds of tags opens the site to XSS. IMO, this should not be allowed and we display what the users entered.